create-oci.sh

```diff
@@ -64,6 +64,12 @@ repo="$(echo -n $store | sed 's_/\(.*\):\(.*\)_/\1_g')"
 for artifact_pair in "${artifact_pairs[@]}"; do
     result_path="${artifact_pair/=*}"
     path="${artifact_pair/*=}"
+
+    if [ -f "${path}/.skip-trusted-artifacts" ]; then
```

Comment:
Does this assume the path is a directory? That's not always the case, right?

Same comment for the `use*` scripts.

This is probably ok. If the path is not a directory, then it just means the skip feature can't be used. It should still work as expected.

Reply:
> Does this assume the path is a directory? That's not always the case, right?

Destination is always a directory, what am I missing?

Reply:
For the create-oci.sh script, `path` is the thing that will be turned into a trusted artifact. Usually, that's a directory, e.g. a local git clone dir. But it could also be a single file, right?

Maybe I'm missing something! :sweat_smile:

Reply:
Oh, right I was confused, yeah `path` could point to a file, I'm guessing that use case is not something we should be concerned with here.